man/download_file.Rd

```diff
@@ -14,6 +21,10 @@ download_file(dataset, logical_path, target, ...)
 \item{target}{A character string representing the location where the file should be downloaded.}
 
 \item{...}{Extra parameters to pass to the download.file() function.}
+
+\item{branch}{The dataset branch.}
```

Comment:
Is it R convention to have the new parameters (branch, transaction) come after "extra" parameters because they're optional?

Reply:
In this case I just thought we might reorder them in the future (e.g. add other parameters), so we'd require users to name their parameters - but I think it's fine to just keep the convention of branch/transaction as being the first 2 optional arguments